DemoApp/DemoApp/TMButtonPage.xaml

```diff
@@ -4,189 +4,129 @@
                     Command="{Binding MyCommand}"
-                    IconSource="icon.png"
-                 Padding="10"
                     CommandParameter="Button5"
-                    Size="Small"
                   />
-                <Modus:TMButton 
-                    x:Name="ibutton3"
-                    Title = "Default"
-                    Command="{Binding MyCommand}"
+                    <Modus:TMButton 
+                    x:Name="button2i"
+                    Title = "Secondary"
+                    ButtonColor="Secondary"
                     IconSource="icon.png"
-                    Padding="10"
-                    CommandParameter="Button5"
-                    Size="Default"
-                  />
-                <Modus:TMButton 
-                    x:Name="ibutton4"
-                    Title = "Large"
-                    Padding="10"
                     Command="{Binding MyCommand}"
-                    IconSource="icon.png"
                     CommandParameter="Button5"
-                   
                   />
-                </FlexLayout>
-                <Label Text="Floating Buttons"/>
-                <FlexLayout Margin="0,0,0,10">
-                <Modus:TMButton 
-                    x:Name="fltbutton2"
-                    Radius="50"
+                </StackLayout>
+                <StackLayout Orientation="Horizontal" Spacing="10">
+                    <Modus:TMButton 
+                    x:Name="button3i"
+                    Title = "Tertiary"
+                    ButtonColor="Tertiary"
+                    IconSource="icondark.png"
                     Command="{Binding MyCommand}"
-                    IconSource="icon.png"
-                    Padding="10"
-                    IsFloatingButton="True"
                     CommandParameter="Button5"
-                    Size="XSmall"
                   />
-
-                <Modus:TMButton 
-                    x:Name="fltbutton"
-                    Title = "XSmall"
-                    Command="{Binding MyCommand}"
+                    <Modus:TMButton 
+                    x:Name="button4i"
+                    Title = "Danger"
                     IconSource="icon.png"
-                    Padding="10"
-                    IsFloatingButton="True"
-                    CommandParameter="Button5"
-                    Size="XSmall"
-                  />
-                </FlexLayout>
-                <Label Text="Disabled Buttons"/>
-                <FlexLayout Margin="0,0,0,10">
-                <Modus:TMButton 
-                    x:Name="button413"
-                    Title = "Large"
-                    Padding="10"
-                    IsDisabled="True"
+                    ButtonColor="Danger" 
                     Command="{Binding MyCommand}"
                     CommandParameter="Button5"
-                    Size="Large"
                   />
+                </StackLayout>
+
 
+                <StackLayout Orientation="Horizontal">
```

Comment:
Can we move this radio button UI to a separate page? Since this page is maintained for buttons?

Reply:
The radio button is used to set properties of the button and set it dynamically. Do you want the radio button UI in a separate page and call the page in between this page?

Reply:
Thanks for the info. Its okay to be in same page